test/functionalTest/cases/719_servicepath_for_registrations/cm_forwards_cb_with_servicePath.test

```diff
@@ -0,0 +1,456 @@
+# Copyright 2015 Telefonica Investigacion y Desarrollo, S.A.U
```

Comment:
I have the feeling that the behaviour regarding step 15 may change after "the big refactoring". Howeve, let's keep the .test as it is now.

NTC.